### ISSUE TITLE
Search Query will not get the field value when index field type is bytes.

### ISSUE THREAD

**Issue by geek-frio** [quickwit-oss/quickwit#2039]
**Describe the bug**
Quickwit version: 0.3.1
Query will not get the field value when index field type is bytes.

**Steps to reproduce (if applicable)**

1. Config a index that has bytes field.
2. Ingest data in the index
3. search index value like this: quickwit index search --index xxx --query "*"
4. The bytes field value will not returned.

After reviewing the code, I find the problem is occured here:

![图片](https://user-images.githubusercontent.com/40055215/193187804-8ecc861a-6951-4692-a498-d5cca91d1321.png)

we got a json array type value from tantivy, but the field type is set to json string type when do config mapping operation.

![图片](https://user-images.githubusercontent.com/40055215/193188233-26fc525c-79be-4b0c-8e88-2226debf950a.png)
 

**Expected behavior**
Return the base64 encoded field value.

**Configuration:**
Please provide:
quickwit version: 0.3.1

**Comment by fmassot** [comment]
Thanks @geek-frio for the report.

I just opened an [issue](https://github.com/quickwit-oss/tantivy/issues/1576) on tantivy side as documents are first serialized into json at the leaf search level when we call `fetch_docs` function. At this level, we just used tantivy `schema.to_json` method to serialize a tantivy document.

It is only at the root level that we use the doc mapper to convert a flat json object into a json object that follows the structure described in the doc mapping.
